packages/instantsearch.js/src/lib/__tests__/status.test.ts

```diff
@@ -185,7 +185,6 @@ describe('status', () => {
   });
 
   test('lets users render on error with the `render` event', async () => {
-    // expect.assertions(4);
```

Comment:
this should have been removed in #5127 already, but I didn't see the review then